openadapt/config.py

```diff
@@ -134,9 +135,11 @@ class SegmentationAdapter(str, Enum):
 
     # Record and replay
     RECORD_WINDOW_DATA: bool = True
-    RECORD_READ_ACTIVE_ELEMENT_STATE: bool = False
+    RECORD_BROWSER_DATA: bool = True
+    RECORD_READ_ACTIVE_ELEMENT_STATE: bool = True
     RECORD_VIDEO: bool
     RECORD_AUDIO: bool
+    RECORD_BROWSER_EVENTS: bool
```

Comment:
Why do we have both `RECORD_BROWSER_DATA` and  `RECORD_BROWSER_EVENTS`? Please remove one if it is unused. (I think the latter name is more appropriate.)

Reply:
Done here: 995fc5539a338b9d9e1a82ca2bff143d594701d2